### PR TITLE
Remove README.rdoc

### PR DESCRIPTION
**Pull request by soedar**
This file is not really needed since we have README.md

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,3 @@
 # foosball-leaderboard
 Who's the best at foosball?
+Soedar is the best
```
Comment (chesterhow) [inline README.md]: Hmmmmmm

Comment (fonglh) [inline README.md]: I think the name here is wrong. :stuck_out_tongue:

Comment (weiqingtoh) [inline README.md]: At being the worst

**Comment by weiqingtoh** [comment]
Pls squash commits

**Comment by luoyanjie** [comment]
Sai understand

**Comment by soedar** [comment]
okay sry kthxbye

**Pull request closed without merging**